11-payment-encoding.md

```diff
@@ -4,36 +4,53 @@ A simple, extensible QR-code-ready protocol for requesting payments
 over Lightning.
 
 # Table of Contents
+
+  * [Encoding Overview](#encoding-overview)
+    * [Requirements](#requirements)
+  * [Human-Readable Part](#human-readable-part)
+    * [Requirements](#requirements-1)
+    * [Rationale](#rationale)
```

Comment:
ToC sections for other BOLTs do not include 'Requirements' and 'Rationale' sections

Reply:
Thanks.